app.test.js

```diff
@@ -0,0 +1,76 @@
+const { randomFillSync } = require('crypto');
+const Application = require('./app');
+
+describe('Application', () => {
+  it('returns the correct scores for all rounds if there are no strikes and spares', () => {
+    let i = -1;
+    const fakeRollInput = {
+      // returns a the next number in the array every time it is called
+      getRoll: () => {
+        const rolls = [3, 5, 6, 3, 0, 0, 2, 3, 7, 2, 5, 4, 6, 1, 0, 2, 2, 3, 5, 1];
+        i++;
+        return rolls[i];
+      }
+    }
```

Comment:
I really like this bit and how you put all rolls in one variable and then assigned the rolls